docs/source/interfaces/indirect/Indirect Elwin & Iqt.rst

```diff
@@ -18,13 +18,19 @@ Mantid data search paths that contains the sample environment information the
 ELF workspace can also be normalised to the lowest temperature run in the range
 of input files.
 
-.. interface:: Data Analysis
+.. image:: ../../images/Inelastic_Data_Analysis_GUI.png
   :width: 450
-  :widget: tabElwin
 
 Elwin Options
 ~~~~~~~~~~~~~
 
+File or Workspace
+  Choose to download data from a file or a workspace by using this dropdown menu. See image below for demonstration of how to load files using either option.
```

Comment:
Is `download` the correct word here? I would have thought just 'load' was better but I'm just checking!

Reply:
You're right it should be load!